### ISSUE TITLE
Gazebo on Windows: conda-forge installation

### ISSUE THREAD

**Issue by cogneos** [osrf/gazebo#3198]
After a seemingly successful installation, gazebo fails to launch w/ the following error message
![screen](https://user-images.githubusercontent.com/102249338/159772142-a3ac6f6b-a52d-468a-8ae9-fb40191c849b.jpg)

**Comment by traversaro** [comment]
Can you report the output of conda list command?

**Comment by cogneos** [comment]
> Can you report the output of conda list command?

Here is goes:

# packages in environment at C:\Users\gorokhov\Programs\MiniConda:
#
# Name                    Version                   Build  Channel
blosc                     1.21.0               h0e60522_0    conda-forge
boost-cpp                 1.74.0               h9f4b32c_7    conda-forge
brotlipy                  0.7.0           py39h2bbff1b_1003  
bzip2                     1.0.8                h8ffe710_4    conda-forge
ca-certificates           2021.10.8            h5b45459_0    conda-forge
cairo                     1.16.0            h15b3021_1010    conda-forge
certifi                   2021.10.8        py39hcbf5309_1    conda-forge
cffi                      1.15.0           py39h2bbff1b_0  
cfitsio                   4.0.0                hd67004f_0    conda-forge
charset-normalizer        2.0.4              pyhd3eb1b0_0  
conda                     4.12.0           py39hcbf5309_0    conda-forge
conda-content-trust       0.1.1              pyhd3eb1b0_0  
conda-package-handling    1.7.3            py39h8cc25b3_1  
console_bridge            1.0.2                h5362a0b_0    conda-forge
console_shortcut          0.1.1                         4  
cppzmq                    4.7.1                h50e1b43_1    conda-forge
cryptography              36.0.0           py39h21b164f_0  
curl                      7.82.0               h789b8ee_0    conda-forge
dirent                    1.21                          0    conda-forge
dlfcn-win32               1.3.0                h0e60522_0    conda-forge
eigen                     3.4.0                h2d74725_0    conda-forge
expat                     2.4.7                h39d44d4_0    conda-forge
ffmpeg                    4.3.1                ha925a31_0    conda-forge
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.13.96              hce3cb01_2    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
freeimage                 3.18.0               hfcb8cd7_7    conda-forge
freetype                  2.10.4               h546665d_1    conda-forge
freexl                    1.0.6                ha8e266a_0    conda-forge
gazebo                    11.10.2              h6d7d6d0_0    conda-forge
geos                      3.10.2               h39d44d4_0    conda-forge
geotiff                   1.7.0                h38b14a8_7    conda-forge
gettext                   0.19.8.1          ha2e2712_1008    conda-forge
gts                       0.7.6                h7c369d9_2    conda-forge
hdf4                      4.2.15               h0e5069d_3    conda-forge
hdf5                      1.12.1          nompi_h2a0e4a3_104    conda-forge
icu                       69.1                 h0e60522_0    conda-forge
idna                      3.3                pyhd3eb1b0_0  
ilmbase                   2.5.5                h12d4b20_0    conda-forge
jbig                      2.1               h8d14728_2003    conda-forge
jpeg                      9e                   h8ffe710_0    conda-forge
jsoncpp                   1.9.5                h2d74725_1    conda-forge
jxrlib                    1.1                  h8ffe710_2    conda-forge
kealib                    1.4.14               h8995ca9_3    conda-forge
krb5                      1.19.3               h1176d77_0    conda-forge
lcms2                     2.12                 h2a16943_0    conda-forge
lerc                      3.0                  h0e60522_0    conda-forge
libclang                  13.0.1          default_h81446c8_0    conda-forge
libcurl                   7.82.0               h789b8ee_0    conda-forge
libdeflate                1.10                 h8ffe710_0    conda-forge
libffi                    3.4.2                h8ffe710_5    conda-forge
libflang                  5.0.0           h6538335_20180525    conda-forge
libgdal                   3.4.2                h7f66a12_0    conda-forge
libglib                   2.70.2               h3be07f2_4    conda-forge
libiconv                  1.16                 he774522_0    conda-forge
libignition-cmake2        2.11.0               h0e60522_0    conda-forge
libignition-common3       3.13.2               h9bd54ec_1    conda-forge
libignition-fuel-tools4   4.4.0                h98d6f7e_5    conda-forge
libignition-math6         6.10.0               h0e60522_0    conda-forge
libignition-msgs5         5.7.0                h5897182_2    conda-forge
libignition-tools1        1.0.0                h21ff451_0    conda-forge
libignition-transport8    8.1.0                h7435cc3_4    conda-forge
libkml                    1.3.0             h9859afa_1014    conda-forge
libnetcdf                 4.8.1           nompi_h1cc8e9d_101    conda-forge
libode                    0.16.2               h415ef7b_9    conda-forge
libopenblas               0.3.18          pthreads_hc469a61_0    conda-forge
libpng                    1.6.37               h1d00b33_2    conda-forge
libpq                     14.2                 hfcc5ef8_0    conda-forge
libprotobuf               3.19.4               h7755175_0    conda-forge
libraw                    0.20.2               hee1bdec_1    conda-forge
librttopo                 1.1.0                hb1df466_9    conda-forge
libsdformat               9.3.0                h0e60522_5    conda-forge
libsodium                 1.0.18               h8d14728_1    conda-forge
libspatialite             5.0.1               h36c16d9_15    conda-forge
libssh2                   1.10.0               h680486a_2    conda-forge
libtiff                   4.3.0                hc4061b1_3    conda-forge
libusb                    1.0.25             h8ffe710_100    conda-forge
libwebp-base              1.2.2                h8ffe710_1    conda-forge
libxml2                   2.9.12               hf5bbc77_1    conda-forge
libzip                    1.8.0                hfed4ece_1    conda-forge
libzlib                   1.2.11            h8ffe710_1014    conda-forge
llvm-meta                 5.0.0                         0    conda-forge
lz4-c                     1.9.3                h8ffe710_1    conda-forge
menuinst                  1.4.18           py39h59b6b97_0  
ogre                      1.10.12              hadbb816_7    conda-forge
openal-soft               1.21.1               h2d74725_0    conda-forge
openblas                  0.3.18          pthreads_h543f93c_0    conda-forge
openexr                   2.5.5                hab3b255_0    conda-forge
openjpeg                  2.4.0                hb211442_1    conda-forge
openmp                    5.0.0                    vc14_1    conda-forge
openssl                   1.1.1n               h8ffe710_0    conda-forge
pcre                      8.45                 h0e60522_0    conda-forge
pip                       21.2.4           py39haa95532_0  
pixman                    0.40.0               h8ffe710_0    conda-forge
poppler                   22.01.0              h24fffdf_2    conda-forge
poppler-data              0.4.11               hd8ed1ab_0    conda-forge
postgresql                14.2                 h1c22c4f_0    conda-forge
powershell_shortcut       0.0.1                         3  
proj                      9.0.0                h1cfcee9_1    conda-forge
pugixml                   1.11.4               h0e60522_0    conda-forge
pycosat                   0.6.3            py39h2bbff1b_0  
pycparser                 2.21               pyhd3eb1b0_0  
pyopenssl                 21.0.0             pyhd3eb1b0_1  
pysocks                   1.7.1            py39haa95532_0  
python                    3.9.7                h6244533_1  
python_abi                3.9                      2_cp39    conda-forge
pywin32                   302              py39h827c3e9_1  
qt                        5.12.9               h556501e_6    conda-forge
qwt                       6.2.0                hc28b1ea_2    conda-forge
requests                  2.27.1             pyhd3eb1b0_0  
ruamel_yaml               0.15.100         py39h2bbff1b_0  
sdl2                      2.0.18               h0e60522_0    conda-forge
setuptools                58.0.4           py39haa95532_0  
simbody                   3.7                  hade3207_3    conda-forge
six                       1.16.0             pyhd3eb1b0_0  
sqlite                    3.37.0               h2bbff1b_0  
tbb                       2021.5.0             h2d74725_0    conda-forge
tbb-devel                 2021.5.0             h2d74725_0    conda-forge
tiledb                    2.7.1                h95dad36_0    conda-forge
tiny-process-library      2.0.4                h0e60522_0    conda-forge
tinyxml                   2.6.2                h2d74725_2    conda-forge
tinyxml2                  9.0.0                h0e60522_2    conda-forge
tqdm                      4.62.3             pyhd3eb1b0_1  
tzdata                    2021e                hda174b7_0  
urdfdom                   2.3.5                h2d74725_0    conda-forge
urdfdom_headers           1.0.5                h5362a0b_2    conda-forge
urllib3                   1.26.7             pyhd3eb1b0_0  
vc                        14.2                 h21ff451_1  
vs2015_runtime            14.27.29016          h5e58377_2  
wheel                     0.37.1             pyhd3eb1b0_0  
win_inet_pton             1.1.0            py39haa95532_0  
wincertstore              0.2              py39haa95532_2  
xerces-c                  3.2.3                h0e60522_4    conda-forge
xz                        5.2.5                h62dcd97_1    conda-forge
yaml                      0.2.5                he774522_0  
zeromq                    4.3.3                h0e60522_3    conda-forge
zlib                      1.2.11            h8ffe710_1014    conda-forge
zstd                      1.5.2                h6255e5f_0    conda-forge
zziplib                   0.13.69              h1d00b33_1    conda-forge

**Comment by traversaro** [comment]
Related reply: https://github.com/conda-forge/gazebo-feedstock/issues/125#issuecomment-1081560181 . 

TL;DR: Have you tried to install gazebo in a new clean environment (i.e. `mamba create -n gazebo -c conda-forge gazebo`) instead of the `base` environment and check if the problem is still there? Thanks!

**Comment by cogneos** [comment]
It looks like the installation succeded now. Thank you!

**Comment by traversaro** [comment]
Great!
